Build/Build.cs

```diff
@@ -81,7 +81,7 @@ class Build : NukeBuild
         });
 
     Target CalculateNugetVersion => _ => _
-        .OnlyWhenDynamic(() => RunAllTargets || HasSourceChanges)
+        .OnlyWhenDynamic(() => RunAllTargets || HasSourceChanges || IsLocalBuild)
         .Executes(() =>
         {
             SemVer = GitVersion.SemVer;
```

Comment:
I testet this with 5.12 and 6.0.2 and both generate the same `SemVer` when on `develop`, so I think this works as intended...